packages/common/src/types.ts

```diff
@@ -27,7 +27,7 @@ type ConsensusConfig = {
 
 export interface ChainConfig {
   name: string
-  chainId: number | bigint
+  chainId: number | string
```

Comment:
This is also a change for serialization and is not yet reflected in the code base.